extensions/epim/src/epim.tsx

```diff
@@ -0,0 +1,533 @@
+import {
+  showToast,
+  Toast,
+  getPreferenceValues,
+  List,
+  ActionPanel,
+  Action,
+  Form,
+  useNavigation,
+  Icon,
+  Color,
+} from "@raycast/api";
+import { execFile } from "child_process";
+import { promisify } from "util";
+import React from "react";
+
+const execFileAsync = promisify(execFile);
+
+interface Preferences {
+  powershellPath: string;
+}
+
```

Comment:
```suggestion
```

You don't need to manually set preference types as this is autogenerated in `raycast-env.d.ts`